starter_code/queries.md

```diff
@@ -4,80 +4,112 @@
 
 ### 1. All the companies whose name match 'Babelgum'. Retrieve only their `name` field.
 
-<!-- Your Code Goes Here -->
+query: {name : "Babelgum"}
+projection: {name : 1}
+sort: 
+skip: 
+limit: 
 
 ### 2. All the companies that have more than 5000 employees. Limit the search to 20 companies and sort them by **number of employees**.
 
-<!-- Your Code Goes Here -->
+query: { number_of_employees : { $gt: 5000 } }
+projection: 
+sort: {number_of_employees: 1}
+skip: 
+limit: 20
 
 ### 3. All the companies founded between 2000 and 2005, both years included. Retrieve only the `name` and `founded_year` fields.
+query: { $and: [ { founded_year : { $gt: 2000 } } , { founded_year : { $lt: 2005} }] }
+projection: {name : 1, founded_year :1 }
+sort: 
+skip: 
+limit: 
 
-<!-- Your Code Goes Here -->
 
 ### 4. All the companies that had a Valuation Amount of more than 100.000.000 and have been founded before 2010. Retrieve only the `name` and `ipo` fields.
 
-<!-- Your Code Goes Here -->
+query : { $and: [ { "ipo.valuation_amount" : { $gt: 100000000 } } , { founded_year : { $lt: 2010} }] }
+projection : {name : 1,  ipo :1 }
+
 
 ### 5. All the companies that have less than 1000 employees and have been founded before 2005. Order them by the number of employees and limit the search to 10 companies.
 
-<!-- Your Code Goes Here -->
+query : { $and: [ { founded_year : { $lt: 2005 } } , { number_of_employees : { $lt: 1000} }] }
+sort : { number_of_employees :1};
+limit: 10
 
 ### 6. All the companies that don't include the `partners` field.
 
-<!-- Your Code Goes Here -->
+query : { partners : {$exists : false}  }
 
 ### 7. All the companies that have a null type of value on the `category_code` field.
 
-<!-- Your Code Goes Here -->
+query : { category_code : null  }
 
 ### 8. All the companies that have at least 100 employees but less than 1000. Retrieve only the `name` and `number of employees` fields.
 
-<!-- Your Code Goes Here -->
+query : { $and: [{number_of_employees : { $gt :100}} , {number_of_employees : { $lt :1000}} ]  }
+projection : {name :1, number_of_employees : 1}
```

Comment:
it should be  {number_of_employees : { $gte :100}} since it's at least 100 employees.